src/libs/OptionsListUtils.js

```diff
@@ -158,6 +158,7 @@ function getOptions(reports, personalDetails, draftComments, activeReportID, {
     sortByLastMessageTimestamp = false,
     searchValue = '',
     showChatPreviewLine = false,
+    showEmptyRecentReports = false,
```

Comment:
"empty" is maybe a bit ambiguous. Perhaps better to make this `showReportsWithNoComments` so that it is more clear when we check this:

```js
showReportsWithNoComments && hasNoComments && ...
```

Reply:
That's definetly clearer. 
Done!